example/main.tf

```diff
@@ -0,0 +1,22 @@
+provider "aws" {
+  region = "eu-west-1"
+}
+
+module "example_team_s3" {
+  source = "../"
```

Comment:
Please use an absolute URI here, this would not work for anyone trying to replicate the example outside this repository.

Reply:
done